doc/adr/release.md

```diff
@@ -0,0 +1,1019 @@
+# Release tool
+
+Create an executable that provides high-level and content-focused controls for
+creating releases from repositories using IOGX.
+
+## Motivation
+
+For engineers focused on the development and maintenance of a product, releases
+are only a means to distribute their works in a way that they are strongly
+associated with an identifier that is fit uses when communicating with users
+(typically a version number).
+
+For these engineers, any actions they have to take or decisions they have to
+make, that don't provide real control over what works are distributed in
+releases and when releases are made, are complexities they face in pursuit of
+their goal.
+
+To manage these complexities engineers require knowledge that is very irrelevant
+to the products they support and other complexities the manage frequently.
+
+We seek to remove those complexities from those engineers' workflows while
+providing controls useful for controlling what works are distributed in
+releases.
+
+## Exclusions
+
+### Release Scheduling
+
+There is a need for better solutions for release scheduling, but this ADR will
+focus on providing a solution that integrates well with the tools and processes
+the smart contracts teams are already using to schedule release, mainly Github
+Actions and manually creating releases.
+
+## Current state
+
+Currently, iogx does not provide any utilities for users to create software
+releases. This leaves users to manually create software releases, document
+complex releases processes and create custom release automation configurations
+and scripts, or get assistance from SCDE to do so.
+
+Not all repositories create any notion of a release, but the ones that do,
+preform releases with a few methods.
+
+- Manually preform specific actions to create releases according to specific
+desired outcomes.
+- Manually preform specific actions to create releases according to a notion of
+convention across IOG or wider contexts.
+- Manually preform specific actions to create releases according to documented
+processes.
+- Creating custom release scrips and manually invoking them.
+- Configuring GitHub actions to automatically invoked custom release scripts on
+specific events and schedules and under specific conditions.
+
+The manually processes are prone to errors and in-material inconsistencies from
+release to release. This is because manually creating release is a processes
+that is contains many decisions and processes and is typically preformed
```

Comment:
is contains -> contains

Reply:
Done c6b3fd08c80b16e0eedc15972bf37468be4e6ed1